app/server/lib/DocStorage.ts

```diff
@@ -1568,6 +1570,33 @@ export class DocStorage implements ISQLiteDB, OnDemandStorage {
     return this._markAsChanged(db.requestVacuum());
   }
 
+  /**
+   * Run a VACUUM on the document and mark it as changed only
+   * if the saved space is above the ratio defined in SHRINK_RATIO_FOR_PUSH.
+   * Therefore if the doc storage is a remote one (like S3), it would be pushed
+   * only when we meet this condition, otherwise it is assumed not to be necessary.
+   */
+  public async vacuum(): Promise<void> {
+    const db = this._getDB();
+    const initSize = await this.storageManager.getFsFileSize(this.docName);
+    log.info(`Start Vacuum of doc ${this.docName}`);
+    // FIXME: Remove it. Adding it solves errors in tests we don't understand yet.
+    await db.exec("SELECT 1");
+
```

Comment:
These lines are meant to test whether the CI pass. Should be removed and an explanation should be found.

```suggestion
```